.editorconfig

```diff
@@ -1,5 +1,5 @@
 [*]
-indent_style = space
+indent_style = tabs
 indent_size = 2
 end_of_line = lf
 charset = utf-8
```

Comment:
I believe this file can actually be removed; modifying it has no impact on the Prettier output, and following the documentation steps don't seem to work as described https://prettier.io/docs/en/configuration#editorconfig

Reply:
confident that file is there for the owners' own convenience, not for Prettier. it must work for whatever IDE they prefer?
better to leave it alone, and ask them whether it can be removed first